exercises/practice/two-fer/test-two-fer.bats

```diff
@@ -37,21 +24,21 @@ teardown() {
 
 @test "empty file given" {
   [[ $BATS_RUN_SKIPPED == "true" ]] || skip
-  run gawk -f two-fer.awk empty.txt
+  run gawk -f two-fer.awk <<< ""
   assert_success
   assert_output "One for you, one for me."
 }
 
 @test "a name given" {
   [[ $BATS_RUN_SKIPPED == "true" ]] || skip
-  run gawk -f two-fer.awk one.txt
+  run gawk -f two-fer.awk <<< "Alice"
   assert_success
   assert_output "One for Alice, one for me."
 }
 
 @test "multiple names given" {
   [[ $BATS_RUN_SKIPPED == "true" ]] || skip
-  run gawk -f two-fer.awk two.txt
+  run gawk -f two-fer.awk <<< "Mary Ann"
```

Comment:
I wanted this one to have multiple lines

Reply:
Reverted.